codegen.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  schema: "https://graphql-pokemon2.vercel.app/",
+  documents: ["./**/*.graphql"],
+  generates: {
+    "./generated/types.ts": {
```

Comment:
This is the file where the generated code will end up (remember that it's gitignored)